.gitignore

```diff
@@ -0,0 +1,90 @@
+# Xcode
```

Comment:
this file is unrelated to the rest of the PR, but it will prevent us from checking in files that don't need to be in the repo.